### ISSUE TITLE
Update Data Download page

### ISSUE THREAD

**Issue by mradamcox** [healthyregions/oeps#32]
With the v2 release we will have new ways to download and access the OEPS data. Here are some updates to the /download page we'll need to make.

- [x] Change header to Data Download & Access
- [x] Change the link buttons to be the following:
    - Title: Download all data, Subtitle: CSVs and Data Dictionaries, Link: (empty for now)
    - Title: Download all data, Subtitle: Full Github archive, Link: (use existing link to Github v1 release ZIP file)
    - Title: Download all data, Subtitle: Frictionless Data Package (v1), Link: (empty for now)
    - Title: Google BiqQuery, Subtitle: Request access key, Link: https://docs.google.com/forms/d/e/1FAIpQLSfTfrxpCoXeHEJMZ9mwRLNlFkWCxLzew_1ayEofK5W59VNKMw/viewform?usp=sf_link
 - [x] Implement year-based selection similar to existing theme-based selection (@bucketteOfIvy has already done this)

Note: Currently the download page has zip and tarball compressed formats, linking directly to the archive files that Github creates with each release. I don't see any need to provide the tarball (it is 503mb where the zip is 504mb, i.e. no difference here in compression ratio).

We will likely have some extra text description to this page, but that can come later.

**Comment by mradamcox** [comment]
Also, set these buttons to be horizontally listed on desktop, and vertical (as they currently are) on mobile.

**Comment by mradamcox** [comment]
Looks great.

**Comment by mradamcox** [comment]
I've completed this by adding new sections for the new download options (including the R package and BigQuery tables), and direct links to all of the consolidated CSV files that we store the datasets in now. Because everything is merged by geography and year, there isn't the same need as before to custom bundle multiple themes into a single download file. So, because of that, and for the sake of expediency, I've just commented out all of the filtering and download ZIP archive generation and replaced it with HTML tables with direct download links.

In the future, we will need to reimplement this download bundling to group metadata markdown files with the merged CSVs--I just wasn't able to get to that this time around.
